aws-billing/aws-billing.json

```
@@ -522,6 +451,6 @@
   },
   "timezone": "browser",
   "title": "AWS Billing",
-  "uid": "AWSBillig",
-  "version": 5
+  "uid": "AWSBilling",
```

I would keep original uid - uid change can make a conflicts for users with old dashboard (because names are the same). Technically, dashboard uid is now UUID. This was just manually created uid for lazy me at that time. I would rather keep it stable with typo. I don't want to cause a problems for users.

---

```suggestion
  "uid": "AWSBillig",
```

Okay, didn't know that. Thanks for your great explanation 😍